src/Compilers/Core/Portable/Operations/IInstanceReferenceExpression.cs

```diff
@@ -16,6 +16,7 @@ namespace Microsoft.CodeAnalysis.Operations
     /// </remarks>
     public interface IInstanceReferenceOperation : IOperation
     {
+        InstanceReferenceKind ReferenceKind { get; }
```

Comment:
>InstanceReferenceKind ReferenceKind { get; } [](start = 8, length = 44)

I would expect to see corresponding changes in two test IOperation visitors and baselines of many existing IOperation tests affected.